devServer.js

```diff
@@ -13,7 +13,7 @@ app.use(require('webpack-dev-middleware')(compiler, {
 
 app.use(require('webpack-hot-middleware')(compiler));
 
-app.get('/', function(req, res) {
```

Comment:
For me the issue persisted until I restarted the server, the hortswap
wasn't enough.
On Mon, Apr 18, 2016 at 8:14 AM Sanjuro Jogdeo <notifications@github.com>
wrote:

> In devServer.js
> <https://github.com/hackoregon/btc-frontend-2016-ReactRedux/pull/31#discussion_r60076607>
> :
>
> > @@ -13,7 +13,7 @@ app.use(require('webpack-dev-middleware')(compiler, {
> >
> >  app.use(require('webpack-hot-middleware')(compiler));
> >
> > -app.get('/', function(req, res) {
>
> And I just realized it's still not working for me. If I got to search
> page, search successfully, and then try to reload the search page, I get a
> "Cannot GET/search" error. In general, problems seem to happen on reload.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/hackoregon/btc-frontend-2016-ReactRedux/pull/31/files/3803c7ca23f49e8a6fac025576a2557a286ba73c#r60076607>
>